virtool/uploads/db.py

```diff
@@ -4,12 +4,64 @@
 from sqlalchemy import select, update
 from sqlalchemy.ext.asyncio import AsyncSession, AsyncEngine
 
+import virtool.tasks.task
 import virtool.uploads.utils
 import virtool.utils
+
+from virtool.types import App
 from virtool.uploads.models import Upload
 
 logger = logging.getLogger("uploads")
 
+PROJECTION = [
```

Comment:
Shouldn't be used anywhere as Mongo isn't used anymore for this data.

Reply:
@igboyes We still need `projection` [here](https://github.com/virtool/virtool/blob/f74a5f35edb9947856618cabf3f00c931c071b0d/virtool/db/core.py#L327), the `MigrateFilesTask` still need to use `files` collection in Mongo.